backend/src/controller/Boards/Boards.controller.js

```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import { BoardsService } from '../../service/Boards/BoardsService.js';
+import { Board } from '../../service/Boards/Board.js';
+
+export function BoardsController() {
```

Comment:
```suggestion
export function boardsController() {
```
`funkcja` więc z małej literki - `klasy` z dużej :)